doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -94,6 +94,14 @@ Matter
   You can still build all examples with deprecated Mbed TLS support by setting the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA` Kconfig option to ``n``, but you must build the Thread libraries from sources.
   To :ref:`inherit Thread certification <ug_matter_device_certification_reqs_dependent>` from Nordic Semiconductor, you must use the PSA Crypto API backend.
 * Updated the page about :ref:`ug_matter_device_low_power_configuration` with the information about Intermittently Connected Devices (ICD) configuration.
+* Updated the Factory Data Guide and added a new section explaining several important Kconfig options.
+
+* Refactored the factory data default usage:
+
+  * The SPAKE2+ verifier is now generated by default with each build, and it will change if any of the related Kconfig options are modified.
+    This resolves the known issue: KRKNWK-18315.
+  * The Test Certification Declaration can now be generated independently and no longer necessitates the generation of the DAC and PAI certificates.
+  * The Rotating Device ID Unique ID can be utilized and generated only if the :kconfig:option:`CONFIG_CHIP_ROTATING_DEVICE_ID` Kconfig option is set to ``y``.
```

Comment:
```suggestion
* Updated:

   * The :ref:`ug_matter_device_low_power_configuration` page with the information about Intermittently Connected Devices (ICD) configuration.
  * The Factory Data Guide and added a new section explaining several important Kconfig options.

* Changed the factory data default usage:

  * The SPAKE2+ verifier is now generated by default with each build, and it will change if any of the related Kconfig options are modified.
    This resolves the :ref:`known issue <known_issues>` related to the SPAKE2+ verifier not regenerating (KRKNWK-18315).
  * The Test Certification Declaration can now be generated independently and no longer necessitates the generation of the DAC and PAI certificates.
  * The Rotating Device ID Unique ID can be utilized and generated only if the :kconfig:option:`CONFIG_CHIP_ROTATING_DEVICE_ID` Kconfig option is set to ``y``.
```